web/handler/pizzas.ml

```diff
@@ -1,8 +1,49 @@
 let index req =
   let open Lwt.Syntax in
+  let csrf = Sihl.Web.Csrf.find req |> Option.get in
+  let alert = Sihl.Web.Flash.find_alert req in
+  let notice = Sihl.Web.Flash.find_notice req in
   let* user = Service.User.Web.user_from_session req |> Lwt.map Option.get in
-  Lwt.return @@ Sihl.Web.Response.of_html (View.Pizzas.index user)
+  let* pizzas = Pizza.find_pizzas () in
+  Lwt.return
+  @@ Sihl.Web.Response.of_html
+       (View.Pizzas.index user ~alert ~notice csrf pizzas)
+;;
+
+let show req =
+  let open Lwt.Syntax in
+  let csrf = Sihl.Web.Csrf.find req |> Option.get in
+  let alert = Sihl.Web.Flash.find_alert req in
+  let notice = Sihl.Web.Flash.find_notice req in
+  let* user = Service.User.Web.user_from_session req |> Lwt.map Option.get in
```

Comment:
@timohuber Just for future reference, I'm not sure if you've encountered this yet, but `Option.get` forces out a `user` from a `Some user` and fails with an exception if the user in a session is `None`, thus resulting in an HTTP 500 status code. Here it won't throw this exception because to access this route, an authentication middleware is used to ensure the user is actually logged in, so the user in the session will never be `None`. I'm pointing this out to illustrate that this is not always the case 
(Just an FYI in case you were not aware before)